stats/stats/src/charts/lines/new_txns.rs

```diff
@@ -0,0 +1,100 @@
+use crate::{
+    charts::{insert::DateValue, ChartUpdater},
+    UpdateError,
+};
+use async_trait::async_trait;
+use chrono::NaiveDate;
+use entity::sea_orm_active_enums::ChartType;
+use sea_orm::{prelude::*, DbBackend, FromQueryResult, Statement};
+
+#[derive(Default, Debug)]
+pub struct NewTxns {}
+
+#[async_trait]
+impl ChartUpdater for NewTxns {
+    async fn get_values(
+        &self,
+        blockscout: &DatabaseConnection,
+        last_row: Option<NaiveDate>,
+    ) -> Result<Vec<DateValue>, UpdateError> {
+        let stmnt = match last_row {
+            Some(row) => Statement::from_sql_and_values(
+                DbBackend::Postgres,
+                r#"
+                SELECT 
+                    date(b.timestamp) as date, 
+                    COUNT(*)::TEXT as value
```

Comment:
does this count work due to group by?
i.e. it first makes group by, and then counts items in each group?

Reply:
yes, it will work, the same as with new_blocks chart